torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -775,13 +775,13 @@ ProcessGroupNCCL::ProcessGroupNCCL(
       terminateProcessGroup_(false),
       terminateHeartbeatMonitorThread_(false),
       collectiveDebugInfoMode_(false),
-      uid_(process_group_id++),
+      local_uid_(process_group_id++),
```

Comment:
Is local_id better? UID means unique id, it's arguably unique locally but adding the U might make people think it's more globally unique